### DISCUSSION TITLE
12 button keypad questions: (jaguar, 5200, CV. INTV)

### DISCUSSION THREAD

**Discussion by SinisterSticks** [darthcloud/BlueRetro#631] [Q&A]
A pre-purchase question.  How do I get the keypad on the Jaguar to be actuated when I plug in another joystick with no obvious keypad into the Jaguar.  Shouldn't you guys have also included/offered as an option a Bluetooth version of the 12 number keypad.

And speaking of 12 number keypads how do you plan to deal with intellivision (with this 16 ways yet one degree of intensity control disc) and ColecoVision and 5200 and Astrocade?

**Comment by darthcloud** [comment]
No idea what you talking about, you likely on the wrong project page.

**Reply by SinisterSticks** [reply]
Isn't there a Jaguar Bluetooth adapter for modern controllers? 

If so I got the right page.

How do you input the keypad numbers in a Jag2BT adapter?

Also wondering if there are similar plans for an Intellivision, a 5200 and a Colecovision.

**Reply by DoomyDoomer** [reply]
Use this document to reference button mapping:
https://docs.google.com/spreadsheets/d/e/2PACX-1vT9rPK2__komCjELFpf0UYz0cMWwvhAXgAU7C9nnwtgEaivjsh0q0xeCEiZAMA-paMrneePV7IqdX48/pubhtml

**Comment by SinisterSticks** [comment]
Okay I see you have to use other buttons on a standard control to equal the number buttons.  If for example I wanted to have a 12 digit keypad pass through for the destinations being the jaguar. intellivision, ColecoVision, and 5200, is the way you could split the inputs between a fight stick and a 12 digit keypad to be treated as one BT controller?
